internal/p4constants/p4constants.go

```diff
@@ -101,8 +99,10 @@ const (
 	ActionParamPreQosPipeLoadTunnelParamdstAddr                uint32 = 2
 	ActionParamPreQosPipeLoadTunnelParamsport                  uint32 = 3
```

Comment:
Not related to this PR but it looks like camel case is not applied consistently
```suggestion
	ActionParamPreQosPipeLoadTunnelParamDstAddr                uint32 = 2
	ActionParamPreQosPipeLoadTunnelParamSport                  uint32 = 3
```

Reply:
My guess is that `strcase.ToPascal` needs the `_` to generate a uppercase letter.

Reply:
Yes. To generate an uppercase letter there must be a delimiter char (in this case `_`). We can revisit the constant naming generation in future but I see it as a low priority improvement

Reply:
No concern from me. Changing autogened code is usually trivial (search&replace).

Reply:
Are we in a rush to get this merged? I might be wrong, but it seems a quite trivial change that would improve readability (since the scope of the PR seems a general improvement). In general, anywhere we do `+name` we should do `+"_"+name`, or even better, we could modify `emitEntityConstant` to accept a name and a prefix, and prepend the "_" to the name when necessary.

Reply:
Right. Done